frontend/src/metabase/core/components/ColorRangeSelector/ColorRangeSelector.unit.spec.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+
+import { color } from "metabase/lib/colors";
+import ColorRangeSelector from "./ColorRangeSelector";
+
+const DEFAULT_VALUE = [color("white"), color("brand")];
+const DEFAULT_COLORS = [color("brand"), color("summarize"), color("filter")];
+
+interface TestColorRangeSelectorProps {
```

Comment:
nit: this is a bit different to the usual pattern:

```
const setup = () => { 
  const onChange = jest.fn();
  render(<ColorRangeSelector ... onChange={onChange} />);

  return { onChange }; 
});

it("....", () => {
  const { onChange } = setup(); 
});
```

the pattern is also documented in the frontend docs in the repo